microovn/ovn/commands.go

```diff
@@ -59,11 +62,11 @@ func newOvsdbSpec(dbType OvsdbType) (*ovsdbSpec, error) {
 	return dbSpec, err
 }
 
```

Comment:
Function below was changed to be more flexible and allow us to wait on arbitrary database state. Not just `connected`